### PR TITLE
Additional explanation added to example.yml

### PR DESCRIPTION
**Pull request by Intigracy**
tested-stamp

### CODE REVIEW

File: html/changelogs/example.yml [hunk 1]
```diff
@@ -21,16 +21,17 @@
...
-# Your name.  
+# Your name.
 author: N3X15
 
 # Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
 delete-after: True
 
 # Any changes you've made.  See valid prefix list above.
-# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# USE SPACES.  NOT TABS.  SPACES.
```
Comment (clusterfack) [inline html/changelogs/example.yml]: Actually remove this line entirely, it was based on a mistaken perception

File: html/changelogs/example.yml [hunk 2]
```diff
@@ -21,16 +21,17 @@
...
 # SCREW THIS UP AND IT WON'T WORK.
 # Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# If you're using characters such as # ' : or the like, surround the entire change with quotes as shown in the second example. These quotes don't show up on the changelog once it's merged and prevent errors.
 changes: 
 - rscadd: Added a changelog editing system that should cause fewer conflicts and more accurate timestamps.
```
Comment (ghost) [inline html/changelogs/example.yml]: Remove the double spacing here and otherwise

File: html/changelogs/example.yml [hunk 3]
```diff
@@ -21,16 +21,17 @@
...
+# Your name.
 author: N3X15
 
 # Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
 delete-after: True
 
 # Any changes you've made.  See valid prefix list above.
-# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# USE SPACES.  NOT TABS.  SPACES.
 # SCREW THIS UP AND IT WON'T WORK.
```
Comment (ghost) [inline html/changelogs/example.yml]: Make sure people are aware you need - SPACE rscadd etc for it to work  somewhere or another

File: html/changelogs/example.yml (hunk 2 above)
Comment (Intigracy) [inline html/changelogs/example.yml]: That's not even a doublespace.

**Comment by Intigracy** [comment]
Updated.

**Pull request merged**